### DISCUSSION TITLE
Cannot run the spfx sample project

### DISCUSSION THREAD

**Discussion by sowmiyavigneshkumar** [pnp/sp-dev-fx-webparts#3638] [Q&A] [answered]
I'm getting the following error When I try to run the **npm Install** on reach-rss-reader sample spfx project.

npm ERR! code 1
npm ERR! path C:\Users\Sowmiya Vigneshkumar\sp-dev-fx-webparts\samples\react-rss-reader\node_modules\node-sass
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c node scripts/install.js
npm ERR! node:internal/modules/cjs/loader:361
npm ERR!       throw err;
npm ERR!       ^
npm ERR!
npm ERR! Error: Cannot find module 'C:\Users\Sowmiya Vigneshkumar\sp-dev-fx-webparts\samples\react-rss-reader\node_modules\glob\glob.js'. Please verify that the package.json has a valid "main" entry
npm ERR!     at tryPackage (node:internal/modules/cjs/loader:353:19)
npm ERR!     at Function.Module._findPath (node:internal/modules/cjs/loader:566:18)
npm ERR!     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:27)
npm ERR!     at Function.Module._load (node:internal/modules/cjs/loader:778:27)
npm ERR!     at Module.require (node:internal/modules/cjs/loader:1005:19)
npm ERR!     at require (node:internal/modules/cjs/helpers:102:18)
npm ERR!     at Object.<anonymous> (C:\Users\Sowmiya Vigneshkumar\sp-dev-fx-webparts\samples\react-rss-reader\node_modules\true-case-path\index.js:3:12)
npm ERR!     at Module._compile (node:internal/modules/cjs/loader:1101:14)
npm ERR!     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
npm ERR!     at Module.load (node:internal/modules/cjs/loader:981:32) {
npm ERR!   code: 'MODULE_NOT_FOUND',
npm ERR!   path: 'C:\\Users\\Sowmiya Vigneshkumar\\sp-dev-fx-webparts\\samples\\react-rss-reader\\node_modules\\glob\\package.json',
npm ERR!   requestPath: 'glob'
npm ERR! }

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Sowmiya Vigneshkumar\AppData\Local\npm-cache\_logs\2023-03-24T14_02_36_263Z-debug-0.log

Debugging steps followed
1. tried deleting the node modules and package-lock.json and installed npm still I got error.
2. tried downgrading the node.js version to 14.16.1
3. tried upgrading the spfx v1.11.1. to the current version

**Comment by bcameron1231** [comment, marked as answer]
Hi, please review the Compatibility section on the Sample.
https://github.com/pnp/sp-dev-fx-webparts/tree/main/samples/react-rss-reader

This solution requires Node v10.
